xdev_bot/issues.py

```diff
@@ -8,79 +8,74 @@
 
 
 @router.register('issues', action='opened')
-async def issue_opened_event(event, gh, *args, **kwargs):
-    """ Whenever an issue is opened, create a card in
-
-    - project_board = "Backlog Queue"
-    - column = "to_do"
-    """
-    to_do_column_id = PROJECT_BOARD['columns']['to_do']['id']
+@router.register('issues', action='reopened')
+@router.register('issues', action='closed')
+@router.register('pull_request', action='opened')
+@router.register('pull_request', action='closed')
+async def issue_event(event, gh, *args, **kwargs):
     project_board_name = PROJECT_BOARD['name']
-    issue_url = event.data['issue']['html_url']
-    url = f'/projects/columns/{to_do_column_id}/cards'
 
-    print(f'Creating Card in {project_board_name} project board for issue : {issue_url}')
+    if event.data[0] == 'issues':
+        html_url = event.data['issue']['html_url']
+    if event.data[0] == 'pull_request':
+        issue_url = event.data['pull_request']['issue_url']
+        html_url = event.data['pull_request']['html_url']
+    
 
-    # POST /projects/columns/:column_id/cards
-    await gh.post(
-        url, data={'note': issue_url}, accept='application/vnd.github.inertia-preview+json'
-    )
+    if event.data['action'] == 'opened':
+        card_Action = 'Creating'
+        column_id = PROJECT_BOARD['columns']['to_do']['id']
+        url = f'/projects/columns/{column_id}/cards'
 
```

Comment:
The next `if` block does not need to be nested inside the previous block (i.e., it can be at the end).  Fewer `if` block nesting is better.